Fluent/Post_Process.ipynb

```diff
@@ -6,127 +6,63 @@
-      "text/html": [
-       "\n",
-       "        <iframe\n",
-       "            width=\"400\"\n",
-       "            height=\"300\"\n",
-       "            src=\"https://www.youtube.com/embed/kyAJiOXQIq4\"\n",
-       "            frameborder=\"0\"\n",
-       "            allowfullscreen\n",
-       "        ></iframe>\n",
-       "        "
-      ],
-      "text/plain": [
-       "<IPython.lib.display.YouTubeVideo at 0x1041d0f98>"
-      ]
-     },
-     "execution_count": 1,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
-   "source": [
-    "from IPython.display import YouTubeVideo\n",
-    "YouTubeVideo(\"kyAJiOXQIq4\")"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## Write an Equation\n",
+    "User can also visualize the vortex shedding behind the cylinder using the case and date files exported at each time step. These files are loaded into CFD Post and an animation can be created to visualize the vortices. More details can be found [here](https://confluence.cornell.edu/display/SIMULATION/Unsteady+Flow+Past+a+Cylinder+-+Numerical+Results). A YouTube video has been created [here](https://www.youtube.com/watch?v=SrynvmBNA2E). \n",
+    "\n",
```

Comment:
Implement the You Tube video using the code provided in the tutorial. You can send me the video to be uploaded on SFO-project YouTube channel as well. I will update the link later.